lib/ruby_llm/mcp/parameter.rb

```diff
@@ -15,7 +15,7 @@ def initialize(name, type: "string", desc: nil, required: true, default: nil, un
       end
 
       def item_type
-        @items["type"].to_sym
+        @items&.[]("type")&.to_sym
```

Comment:
Hey!

Possible to use dig on this, that's the pattern that used in other locations in the codebase

```ruby
@items&.dig("type")&.to_sym
```